src/figuresClasses.ts

```diff
@@ -1,19 +1,72 @@
-export interface Figure {
+const hackerAlert = new Error('Congratz! You\'ve hacked the system!');
 
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea: () => number,
 }
```

Comment:
relevant interface, but i still think you should use enums for shape and color
```suggestion
enum Shape {
  triangle,
  circle,
  rectangle,
}

enum Color {
  red,
  green,
  blue,
}

export interface Figure {
  shape: Shape,
  color: Color,
  getArea: () => number,
}
```